cli/print/networks.go

```diff
@@ -10,18 +10,18 @@ import (
 )
 
 // Table formatting
-var networksTmplTableHeaderSrc = `ID	NAME	STATUS	CREATED	EXPIRES	OUTBOUND`
+var networksTmplTableHeaderSrc = `ID	NAME	STATUS	CREATED	EXPIRES	POLICY`
 var networksTmplTableRowSrc = `{{ range . -}}
-{{ .ID }}	{{ padding .Name 27	}}	{{ padding (printf "%s" .Status) 12 }}	{{ padding (printf "%s" (localeTime .CreatedAt)) 30 }}	{{if .ExpiresAt.IsZero}}{{ padding "-" 30 }}{{else}}{{ padding (printf "%s" (localeTime .ExpiresAt)) 30 }}{{end}}	{{if eq .Outbound ""}}{{ padding "-" 30 }}{{else}}{{ padding (printf "%s" (.Outbound)) 30 }}{{end}}
+{{ .ID }}	{{ padding .Name 27	}}	{{ padding (printf "%s" .Status) 12 }}	{{ padding (printf "%s" (localeTime .CreatedAt)) 30 }}	{{if .ExpiresAt.IsZero}}{{ padding "-" 30 }}{{else}}{{ padding (printf "%s" (localeTime .ExpiresAt)) 30 }}{{end}}	{{if eq .Policy ""}}{{ padding "-" 30 }}{{else}}{{ padding (printf "%s" (.Policy)) 30 }}{{end}}
```

Comment:
Should "default" Policy be "Open" vs "-"?

Reply:
We set the default policy in the db as `"open"` so it shouldn't _ever_ be null, but we can set it for the sake of having it here.